### PR TITLE
corrected changes

### PR DESCRIPTION
**Pull request by PemaYT**
made the cosmetic changes that I missed earlier :)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-* {
+  * {
```
Comment (Zyanvel) [inline styles.css:1]: I think this is an unnecessary space but you can correct it later, just a heads up.

**Review by Zyanvel — APPROVED** [review]
Everything looks good to me, just left a small nit comment that can be corrected later but it is not a blocker to merge this PR. Just make sure you change it later, good job :)

File: styles.css (hunk 1 above)
Comment (PemaYT) [reply styles.css:1]: thank you

**Comment by soleyingunn** [comment]
Looks good :D

**Pull request merged**